configure.ac

```diff
@@ -548,7 +548,7 @@ AM_CONDITIONAL([FONTTOOLS],[test x"${fontforge_extras}" = xyes])
 # Create and Set FontForge Compiler Flags
 
 my_cflags="${my_cflags} ${WARNING_CFLAGS}"
-my_cflags="${my_cflags} -Werror=implicit-function-declaration -Werror=int-conversion"
```

Comment:
What version of gcc is this?

Reply:
Whatever ships on Ubuntu 14.04.

Reply:
I got the error on Launchpad. I don't have an active Trusty machine anymore.

Reply:
I'd prefer not disabling this, since it is useful, and has caught issues in the past. I'll see if I can modify this to detect if the flag is supported and only add it if it is.

Reply:
Okay.

Reply:
@frank-trampe take a look

Reply:
I like that.

Reply:
Apologies, I didn't realise that AC_MSG_ERROR prevents configure from continuing - I've updated it to not error out in that case (and I've also spun up a trusty VM to test against)